about-us.html

```diff
@@ -65,211 +69,198 @@ <h2 class="section-heading">THE STORY</h2>
+                <img src="images\partners\giving-block.svg" alt="giving block" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="venly">
+                <img src="images\partners\venly.svg" alt="venly" width="162" height="60">
             </div>
-            <div class="partner-logo"><img src="#" alt="pillar">
+            <div class="partner-logo">
+                <img src="images\partners\pillar.svg" alt="pillar" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="dappRadar">
+                <img src="images\partners\dappradar.svg" alt="dappRadar" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="synthetix">
+                <img src="images\partners\synthetix.svg" alt="synthetix" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="unstoppable">
+                <img src="images\partners\unstoppable.svg" alt="unstoppable" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="live coin watch">
+                <img src="images\partners\live-coin-watch.svg" alt="live coin watch" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="bitpay">
+                <img src="images\partners\bitpay.svg" alt="bitpay" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="unstoppable domains">
+                <img src="images\partners\unstoppable-domains.svg" alt="unstoppable domains" width="162" height="60">
             </div>
             <div class="partner-logo">
-                <img src="#" alt="huobi_wallet">
+                <img src="images\partners\huobi-wallet.svg" alt="huobi_wallet" width="162" height="60">
             </div>
         </section>
     </main>
     <footer class="page-footer">
         <section class="catalog-group">
             <div class="join-our-community">
-                <h2 class="joc-heading">JOIN OUR COMMUNITY</h2>
+                <h2 class="joc heading">JOIN OUR COMMUNITY</h2>
                 <p class="joc-text">
                     Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                 </p>
                 <div class="joc-buttons">
-                    <button type="button" class="discord-button">
-                        <img src="path/to/discord-logo.png" alt="discord-logo"> DISCORD
-                    </button>
-                    <button type="button" class="whitelist-button">
-                        <img src="path/to/whitelist-logo.png" alt="whitelist-logo"> WHITELIST NOW
-                    </button>
+                    <button type="button" class="discord-button">DISCORD</button>

```

Comment:
you need to add styles for `discord-button` and `whitelist-button`
use the same classes like in header